samples/connectors/src/main/java/quarks/samples/connectors/iotf/IotfSensors.java

```diff
@@ -109,6 +99,31 @@ public static void simulatedSensors(IotDevice device, boolean print) {
         // with event identifier "sensors".
         device.events(sensors, "sensors", QoS.FIRE_AND_FORGET);
     }
+    
+    /**
+     * Create a heart beat device event with
+     * identifier {@code heartbeat} to
+     * ensure there is some immediate output and
+     * the connection to IoTF happens as soon as possible.
+     * @param device IoT device
+     */
+    public static void heartBeat(IotDevice device, boolean print) {
```

Comment:
Wondering about promoting (unncessary?) heartbeat messages.  While IotDevice isn't necessarily MQTT specific, MQTT has inherent mechanisms for dealing with this sort of thing:  KeepAlive and LastWillAndTestament.  Discussions [here-on-stackoverflow](http://stackoverflow.com/questions/17270863/mqtt-what-is-the-purpose-or-usage-of-last-will-testament) and [here-on-hivemq](http://www.hivemq.com/blog/mqtt-essentials-part-9-last-will-and-testament). Perhaps IotDevice should include an interface for expressing this and impls do what's needed to comply?